src/reanimated2/utils.ts

```diff
@@ -0,0 +1,20 @@
+import { RefObject } from 'react';
+import { measure } from '.';
+
+export interface ComponentCoords {
+  x: number;
+  y: number;
+}
+
+export function convertCoords<T>(
+  parentRef: RefObject<T>,
+  x: number,
+  y: number
+): ComponentCoords {
+  'worklet';
+  const parentCoords = measure(parentRef);
+  return {
+    x: x - parentCoords.x,
+    y: y - parentCoords.y,
+  };
+}
```

Comment:
If I understand correctly. if someone wants to use convertCoords they need to call measure on current component and then pass `x` and `y`? I don't really like this solution. Maybe a more ergonomic solution is passing two refs (current and parent) and do a measure inside the function.